mxnet/training/docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -98,17 +98,17 @@ WORKDIR /
 # python-dateutil==2.8.0 to satisfy botocore
```

Comment:
Note: buildspec is currently set to MX1.8 only, so this PR will only build/test MX 1.8 images

Reply:
Are you suggesting that I can only update MX1.8 in this PR?

Reply:
Hey all, you can add diff versions to buildspec, but if planning on making this change for several version, I'd suggest openings separate PRs for ease of iteration and in order to debug issues separately